### PR TITLE
gstreamer-0-10 (GStreamer 0.10): drop

### PR DESCRIPTION
**Pull request by jiegec**
Topic Description
-----------------

- mugshot: bump REL due to gstreamer-0-10 removal
- qt-4: bump REL due to gstreamer-0-10 removal
    Patches are tracked at AOSC-Tracking/qt4
- libnice: bump REL due to gstreamer-0-10 removal
- gst-plugins-*-0-10: drop
- gstreamer-0-10: drop

Package(s) Affected
-------------------

- libnice: 0.1.17-2
- mugshot: 0.4.1-6
- qt-4: 4.8.7-20

Security Update?
----------------

No

Build Order
-----------

```
#buildit mugshot libnice qt-4
```

Test Build(s) Done
------------------

**Primary Architectures**

- [x] AMD64 `amd64`
- [x] AArch64 `arm64`
- [ ] LoongArch 64-bit `loongarch64`

**Secondary Architectures**

- [x] Loongson 3 `loongson3`
- [x] PowerPC 64-bit (Little Endian) `ppc64el`
- [x] RISC-V 64-bit `riscv64`

**Experimental Architectures**

- [x] MIPS R6 64-bit (Little Endian) `mips64r6el`

### CODE REVIEW

**Review by MingcongBai — APPROVED** [review]

File: runtime-optenv32/wxgtk+32/autobuild/defines [hunk 1]
```diff
@@ -1,6 +1,6 @@
 PKGNAME=wxgtk+32
 PKGSEC=libs
-PKGDEP="gtk-2+32 sdl+32 gst-plugins-base-0-10+32"
+PKGDEP="gtk-2+32 sdl+32"
 PKGDES="GTK+ implementation of wxWidgets API for GUI"
 
 NOLTO=1
```
Comment (MingcongBai) [inline runtime-optenv32/wxgtk+32/autobuild/defines]: Bump REL.

File: runtime-optenv32/wxgtk-3+32/autobuild/defines [hunk 2]
```diff
@@ -1,6 +1,6 @@
 PKGNAME=wxgtk-3+32
 PKGSEC=libs
-PKGDEP="gtk-2+32 sdl+32 gst-plugins-base-0-10+32"
+PKGDEP="gtk-2+32 sdl+32"
 BUILDDEP="32subsystem"
 PKGDES="GTK+ implementation of wxWidgets API for GUI"
```
Comment (MingcongBai) [inline runtime-optenv32/wxgtk-3+32/autobuild/defines]: Bump REL.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: app-utils/mugshot/autobuild/defines [hunk 3]
```diff
@@ -1,6 +1,6 @@
 PKGNAME=mugshot
 PKGSEC=utils
-PKGDEP="accountsservice dbus-python gst-plugins-good-0-10 py3cairo pygobject-3 pexpect"
+PKGDEP="accountsservice dbus-python py3cairo pygobject-3 pexpect"
```
Comment (MingcongBai) [inline app-utils/mugshot/autobuild/defines:3]: Bump REL.

**Review by MingcongBai — CHANGES_REQUESTED** [review]

File: runtime-optenv32/32subsystem/autobuild/defines [hunk 4]
```diff
@@ -7,8 +7,8 @@ at-spi2-core+32 attr+32 binutils+32 bzip2+32 cairo+32 cdparanoia+32 celt+32 cg+3
 curl+32 db+32 dbus+32 dbus-glib+32 elfutils+32 eudev+32 expat+32 faac+32 faad2+32 fftw+32  \
 flac+32 fontconfig+32 freealut+32 freeglut+32 freetype+32 game-music-emu+32 gcc+32 gconf+32  \
 gdbm+32 gdk-pixbuf+32 ghostscript+32 giflib+32 glew+32 glib+32 glibc+32 glu+32 gmp+32 gnutls+32  \
-gst-plugins-bad-1-0+32 gst-plugins-base-0-10+32 gst-plugins-base-1-0+32 gst-plugins-good-1-0+32  \
-gst-plugins-ugly-1-0+32 gstreamer-0-10+32 gstreamer-1-0+32 gtk-2+32 gtk-3+32 harfbuzz+32 icu+32  \
+gst-plugins-bad-1-0+32 gst-plugins-base-1-0+32 gst-plugins-good-1-0+32  \
+gst-plugins-ugly-1-0+32 gstreamer-1-0+32 gtk-2+32 gtk-3+32 harfbuzz+32 icu+32  \
```
Comment (MingcongBai) [inline runtime-optenv32/32subsystem/autobuild/defines]: There should be a version change for this package.

**Review by MingcongBai — DISMISSED** [review]

**Comment by jiegec** [comment]
wxgtk+32 dropped.

**Comment by jiegec** [comment]
Maybe we can drop gstreamer-0-10 first, skipping gstreamer-0-10+32.

**Comment by Cyanoxygen** [comment]
Optenv32 part of this PR will be done in the Wave 2.

**Review by Cyanoxygen — APPROVED** [review]

**Pull request merged**